apps/condo/domains/ticket/components/TicketHint/SettingsContent.tsx

```diff
@@ -0,0 +1,132 @@
+import { PlusCircleOutlined } from '@ant-design/icons'
+import styled from '@emotion/styled'
+import { Col, Row, Typography } from 'antd'
+import { Gutter } from 'antd/es/grid/row'
+import get from 'lodash/get'
+import { useRouter } from 'next/router'
+import React, { useCallback } from 'react'
+
+import { useIntl } from '@core/next/intl'
+
+import Input from '@condo/domains/common/components/antd/Input'
+import { TableFiltersContainer } from '@condo/domains/common/components/TableFiltersContainer'
+import { useSearch } from '@condo/domains/common/hooks/useSearch'
+import { useLayoutContext } from '@condo/domains/common/components/LayoutContext'
+import { DEFAULT_PAGE_SIZE, Table } from '@condo/domains/common/components/Table/Index'
+import { getPageIndexFromOffset, getTableScrollConfig, parseQuery } from '@condo/domains/common/utils/tables.utils'
+import { useOrganization } from '@core/next/organization'
+import { SortTicketHintsBy } from '@app/condo/schema'
+import ActionBar from '@condo/domains/common/components/ActionBar'
+import { Button } from '@condo/domains/common/components/Button'
+import { useQueryMappers } from '@condo/domains/common/hooks/useQueryMappers'
+
+import { useTicketHintTableColumns } from '@condo/domains/ticket/hooks/useTicketHintTableColumns'
+import { useTicketHintTableFilters } from '@condo/domains/ticket/hooks/useTicketHintTableFilters'
+import { TicketHint } from '../../utils/clientSchema'
+import { IFilters } from '../../utils/helpers'
+
+const SORTABLE_PROPERTIES = ['name']
+const TICKET_HINTS_DEFAULT_SORT_BY = ['createdAt_DESC']
+
+const StyledTable = styled(Table)`
+  .ant-table-cell-ellipsis {
+    white-space: inherit;
+  }
+`
+
+const GUTTER_0_40: [Gutter, Gutter] = [0, 40]
```

Comment:
I don't think the name of a style constant should contain its value. If you need to change it in the future, then you will have to change the name of the constant